### PR TITLE
feat: add reversal flag for Card layout

### PR DESCRIPTION
**Pull request by colincclark**
https://nidigitalsolutions.jira.com/browse/REPLAT-1614

- Card now takes a prop to reverse it's flex direction in order for it to work with the new Opinion slice

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/70d8fd02a48dced4ad84895179f212d39284fb1b/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/0fc58f9d4748e9722124d3c83abca70f5a1b8ccd/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/0fc58f9d4748e9722124d3c83abca70f5a1b8ccd/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/33343ab5d8db8c0a43ce71c63f6f4c093aec67d6/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/9c4e414fe432f113f60043384fbc9636eaa3936c/index.html

File: packages/topics/.prettierignore [hunk 1]
```diff
@@ -10,3 +10,4 @@ CHANGELOG.md
 **/coverage/*
 LICENSE
 yarn.lock
+*.log
```
Comment (aronshamash) [inline packages/topics/.prettierignore]: different PR

Comment (colincclark) [reply packages/topics/.prettierignore]: removed from this PR

**Comment by colincclark** [comment]
@jimhunty not sure how to fix this - the styling is controlled within the story, and I am not sure how to switch styles with knobs

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/c5ef887f1d6d157748503e8ba0316b6517f1ac87/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/e26f621d8efff600a0d63ecc93f6bd476bc7644f/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/58e763d7ffe62e461ce595be14a8623d15d198a7/index.html

File: packages/card/card.js [hunk 2]
```diff
@@ -8,12 +8,11 @@ import styles from "./styles";
 
 class CardComponent extends Component {
```
Comment (nikhedonia) [inline packages/card/card.js]: maybe use https://reactjs.org/docs/react-api.html#reactpurecomponent ?

Comment (nikhedonia) [reply packages/card/card.js]: This would make shouldComponentUpdate unnecessary

Comment (colincclark) [reply packages/card/card.js]: Doesn't this just shallow check `state` and `props`? the code rerenders when image url, image size or the isloading state changes, but not image ratio (not sure what the intention was there)

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/a6d64c8dce39d330fa6e618dfc3fb9ebf2413115/index.html

File: packages/card/card.js (hunk 2 above)
Comment (aronshamash) [reply packages/card/card.js]: think this is out of scope for this PR. If you think is worth doing add to github issues.

**Comment by jimhunty** [comment]
Reversed loading bars should not touch the image placeholder. They should have the same margin as the non reversed loader

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/6e1ad1501af82fbbb2e47cf28eef31b17c1086ff/index.html

**Pull request merged**